### ISSUE TITLE
Common Web View 

### ISSUE THREAD

**Issue by AK47-23** [flutter/flutter#140370]
### Use case

I have a common header widget throughout my web app. Now what I'm doing is I am creating a common web view with the headers and child and whatever page I want to show. What I want is, I dont want to call common web view each time while navigating to a new page and creating new instance, instead can I have it only once and only the inner child page changes everytime?

### Proposal

Is it possible in flutter web?

**Comment by dam-ease** [comment]
Hi @AK47-23.
It seems your question is best suited for support channels like StackOverflow, please ask the same there for solutions if any. Please also see [flutterhelp](https://www.reddit.com/r/flutterhelp/) for resources and ask questions like this.

Closing from here as this is more of a how-to question rather than a bug or feature request.
If you are proposing a change, please use the appropriate new issue template and fill out the requested details so that we can address this issue properly.
